inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/gather_elements.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright (C) 2021 Intel Corporation
```

Comment:
This PR: https://github.com/openvinotoolkit/openvino/pull/3822 added ONNX tests for GatherElements which are now disabled due to absence of CPU implementation. Please enable them as part of this PR.

Reply:
ONNX CI fails after that. I don't think we should solve that here.

Reply:
As agreed offline - we need to check is it the implentation issue or not.